### ISSUE TITLE
WSL 2 engine no longer works on 20H1 slow ring

### ISSUE THREAD

**Issue by bushidocodes** [docker/for-win#5169]
- [ ] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Docker Desktop WSL2 engine would work on the new Windows 10 20H1 Slow Ring releases. If this is not possible due to improvements in WSL2, perhaps check the Windows version and warn the user that the update would break WSL2 functionality so they can choose to stay on their current version.

### Actual behavior
The latest version of Docker Desktop Edge increased the minimum version to Windows 10 build 19018, while the slow ring is still on 19013. I did not notice a warning about this, but it's possible I didn't read close enough.

![image](https://user-images.githubusercontent.com/8700322/69103016-33af7200-0a32-11ea-98a0-cf8538973547.png)

### Information

  - Windows Version:
  - Docker Desktop Version:
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM:

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by StefanScherer** [comment]
@bushidocodes Thanks for the feedback. The week before the Slow Ring release we got the 19018 release which unblocked a lot of improvements we had on hold for Docker Desktop. As we saw that Slow ring is got a release 19013 we decided against supporting both the old and new mode in one Docker Desktop release, as it's only a temporary problem. We apologize for the inconvenience. ;-)
Current workaround is to install 2.1.5.0 which you can download from https://docs.docker.com/docker-for-windows/edge-release-notes/ page.
We hope that Slow Ring will get an update to ≥ 19018 soon.

**Comment by StefanScherer** [comment]
Closing this issue as it's a dublicate of #5168
